### PR TITLE
[#5] added Logux.undo shortcut

### PR DESCRIPTION
**Pull request by markfrst**

### CODE REVIEW

File: spec/logux_spec.rb [hunk 1]
```diff
@@ -23,4 +23,27 @@
...
+      }
+    end
+
+    let(:reason) { 'error' }
+
+    before { stub_request(:post, Logux.configuration.logux_host) }
+
+    it 'makes request' do
+      subject
+      expect(WebMock).to have_requested(:post, Logux.configuration.logux_host)
```
Comment (dsalahutdinov) [inline spec/logux_spec.rb]: [Here](https://github.com/wilddima/logux_rails/blob/master/lib/logux/test/helpers.rb#L25) is new implemented matcher to check request is made

File: lib/logux.rb [hunk 2]
```diff
@@ -55,6 +55,16 @@ def self.add(type, meta: {})
     logux_add.call(type, meta: logux_meta)
   end
 
+  def self.undo(meta = {}, reason = nil)
```
[the remark this answers is not in the document]
Comment (markfrst) [reply lib/logux.rb]: @wilddima fixed

File: spec/logux_spec.rb (hunk 1 above)
Comment (markfrst) [reply spec/logux_spec.rb]: @dsalahutdinov fixed

File: lib/logux.rb [hunk 3]
```diff
@@ -55,6 +55,16 @@ def self.add(type, meta: {})
     logux_add.call(type, meta: logux_meta)
   end
 
+  def self.undo(meta = Logux::Meta.new({}), reason = nil)
+    logux_add = Logux::Add.new
+    logux_meta = Logux::Meta.new(meta.merge(status: 'processed'))
```
Comment (wilddima) [inline lib/logux.rb]: @markfrst I asked @ai about it, and we don't need `processed` here, so let's remove this string

File: lib/logux.rb [hunk 4]
```diff
@@ -55,6 +55,16 @@ def self.add(type, meta: {})
     logux_add.call(type, meta: logux_meta)
   end
 
+  def self.undo(meta = Logux::Meta.new({}), reason = nil)
```
Comment (wilddima) [inline lib/logux.rb]: And we lost keyword argument here 😉

File: lib/logux.rb (hunk 3 above)
Comment (markfrst) [reply lib/logux.rb]: okay

File: lib/logux.rb (hunk 4 above)
Comment (markfrst) [reply lib/logux.rb]: which argument?

**Comment by markfrst** [comment]
@wilddima fixed

**Pull request merged**